rucio-server/templates/renew-fts-cronjob.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.ftsRenewal.enabled -}}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-renew-fts-proxy
+spec:
+  schedule: "{{ .Values.ftsRenewal.schedule }}"
```

Comment:
This may cause problems. I assume this is the same schedule as the renewal for the daemon and the reason this is there in the first place is that there is a race condition in FTS where if multiple renewals are attempted simultaneously, the proxy is invalidated.

You should make a separate schedule for the server vs. the daemon renewal.

Reply:
Actually I see this is a separate setting, so it's OK with one caveat below...

Reply:
Thanks for the information. I added a 30m shift in the default schedule.